src/main/kotlin/com/hedvig/rapio/quotes/QuoteServiceImpl.kt

```diff
@@ -182,32 +186,33 @@ class QuoteServiceImpl(
             request.personalNumber
         )
 
-        return when (response) {
-            is Either.Right -> {
-                val completionUrlMaybe: String? = apiGateway.setupPaymentLink(response.b.memberId, response.b.market)
+        return response.bimap(
+            {
+                logger.info("Failed to sign bundle: ${it.errorCode} - ${it.errorMessage}")
+                when (it.errorCode) {
+                    ErrorCodes.MEMBER_BREACHES_UW_GUIDELINES -> "Cannot sign quote, breaches underwriting guidelines"
+                    ErrorCodes.MEMBER_QUOTE_HAS_EXPIRED -> "Cannot sign quote, quote has expired"
+                    ErrorCodes.MEMBER_HAS_EXISTING_INSURANCE -> "Cannot sign quote, already a Hedvig member"
```

Comment:
https://hedvig.atlassian.net/browse/IPC-75
```suggestion
                    ErrorCodes.MEMBER_HAS_EXISTING_INSURANCE -> "Unable to sign quote, Cannot create member"
```